src/main/java/org/spongepowered/api/plugin/Plugin.java

```diff
@@ -55,6 +55,15 @@
     String name();
 
     /**
+     * Gets the prefixes that can be prepended to a command to indicate this
+     * plugin's version of said command. This must have at least one item.
+     * 
+     * @return The prefix prepended to a command to indicate this plugin's
+     *         version of said command.
+     */
+    String[] commandPrefixes();
```

Comment:
@Deamon5550 I guess I'll mention that PluginContainer impls should map getRegistrarName to getId